example/interface.js

```diff
@@ -8,10 +8,19 @@
 //  See the accompanying file LICENSE or http://www.apache.org/licenses/LICENSE-2.0.html
 //
 
-import { Vircadia, DomainServer, AudioMixer, AvatarMixer, MessageMixer, Uuid } from "../dist/Vircadia.js";
+import {
+    Vircadia,
+    DefaultLogConfiguration,
+    DomainServer, AudioMixer,
+    AvatarMixer,
+    MessageMixer,
+    Uuid
+} from "../dist/Vircadia.js";
 
 (function () {
 
+    DefaultLogConfiguration.filter = undefined;
```

Comment:
Delete line because logger isn't used in the example app?

Reply:
Clearing the filter property is required here, since the logger is configured with an error and warning filter by default, to keep the dev console clean for SDK users, but in the example app we want all messages for testing and debugging purposes. This will have an effect once the existing console logs in the SDK are replaced with the logger calls. We can make this part of the build configuration instead, but I wasn't sure how to do it. The usual webpack ways of detecting a production build didn't quite work for me, and I didn't want to change too much in the build scripts in the scope of this PR.